src/main/java/no/unit/nva/cristin/projects/UriUtils.java

```diff
@@ -2,10 +2,31 @@
 
 import static nva.commons.core.attempt.Try.attempt;
 import java.net.URI;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
 
 public class UriUtils {
 
     public static URI buildUri(String... parts) {
         return attempt(() -> new URI(String.join("/", parts))).orElseThrow();
     }
+
+    /**
+     * Creates as string from a map containing a formatted query parameters string to be put on end of an url.
+     *
+     * @param queryParameters The query params
+     * @return a String with query parameters formatted to be placed at end of url
+     */
+    public static String queryParameters(Map<String, String> queryParameters) {
+        return Optional.ofNullable(queryParameters)
+            .map(UriUtils::formatQueryParameters)
+            .orElse(null);
```

Comment:
Call this EMPTY_QUERY_PARAMETERS_FOR_URI_CONSTRUCTOR as it isn't clear why we would return null here rather than say Optional<String>.